doc/nrf/ug_nrf5340.rst

```diff
@@ -589,6 +591,42 @@ Additionally, the memory partitions must be defined and include:
 Samples and applications built for Thingy:53 enable simultaneous update of multiple images by default.
 To learn more about Thingy:53, see :ref:`ug_thingy53`.
 
+MCUboot's serial recovery of the networking core image
+******************************************************
+
+In addition to the recovery of the application core image, also the networking core image can be recovered.
+You can use this feature if you build nRF53 MCUboot with one of the following options:
+
+* :kconfig:option:`CONFIG_NRF53_MULTI_IMAGE_UPDATE` - Simultaneous multi-image DFU.
+* :kconfig:option:`CONFIG_NRF53_RECOVERY_NETWORK_CORE` - Serial recovery for the network core in the one image pair mode.
+
+To upload the networking image, use the following command::
+
+     ./mcumgr image upload <build_dir_path>/zephyr/net_core_app_update.bin -e -n 3 -c serial_conn
+
+``serial_conn`` is the serial connection configuration.
+
+To enable serial recovery of the network core while multi-image update is not enabled in the MCUboot, set the following options:
+
+* select :kconfig:option:`BOOT_IMAGE_ACCESS_HOOK`
+* select :kconfig:option:`FLASH_SIMULATOR`
+* select :kconfig:option:`FLASH_SIMULATOR_DOUBLE_WRITES`
+* disable :kconfig:option:`FLASH_SIMULATOR_STATS`
+* select :kconfig:option:`MCUBOOT_SERIAL_DIRECT_IMAGE_UPLOAD`
+* select :kconfig:option:`NRF53_RECOVERY_NETWORK_CORE`
+
+Additionally, define and include the memory partitions:
+
+* ``mcuboot_primary`` and ``mcuboot_secondary`` - partitions for the application core image slots.
+* ``mcuboot_primary_1`` - partition for the network core image slot.
+* ``pcd_sram`` - partition used for command exchange between the application core and the network core (see :kconfig:option:`CONFIG_PCD_APP`).
+
+.. note::
+
+   When using MCUboot with the :kconfig:option:`CONFIG_NRF53_RECOVERY_NETWORK_CORE` option enabled,
+   the application core does not have direct access to the network core flash memory.
+   Due to this, mcuboot_primary_1 must be used as the RAM partition mediator.
+
 Container for firmware update binaries
 ======================================
```

Comment:
Is this section related to the serial recovery of the network core? It is now set to become a subsection of that chapter.

Reply:
@nvlsianpu Please check Greg's comment

Reply:
@FrancescoSer `Container for firmware update binaries` section is related to `Simultaneous multi-image DFU` and DFU in general.